### ISSUE TITLE
Voicing statement does not match available options

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/gravity-force-lab-basics#302]
**Test device**
Chromebook

**Browser**
Chrome 

**Problem description**
https://github.com/phetsims/qa/issues/702

When Mass Sphere 2 is on last stop right and Mass Sphere 1 is moved to last stop right, if you then click on Sphere 2, Voicing says "Move Mass 2" even though that is not possible. This also occurs in the opposite situation (mass sphere 1 at last stop left).

**Steps to reproduce**
1. Turn on Voicing in preferences
2. Move Sphere 2 all the way to the right
3. Move Sphere 1 all the way to the right
4. Click on Sphere 2

**Visuals**
https://drive.google.com/file/d/19I4dYag7BaONxYoXYBzxwF2ihJYPReD_/view?usp=sharing

Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Gravity Force Lab: Basics‬
URL: https://phet-dev.colorado.edu/html/gravity-force-lab-basics/1.1.0-rc.2/phet/gravity-force-lab-basics_all_phet.html
Version: 1.1.0-rc.2 2021-08-30 16:27:03 UTC
Features missing: applicationcache, applicationcache, touch
User Agent: Mozilla/5.0 (X11; CrOS x86_64 13982.82.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.157 Safari/537.36
Language: en-US
Window: 1357x609
Pixel Ratio: 1/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 32 uniform: 4096
Texture: size: 16384 imageUnits: 32 (vertex: 32, combined: 192)
Max viewport: 16384x16384
OES_texture_float: true

**Comment by jessegreenberg** [comment]
Reviewed in meeting today with @terracoda - @terracoda said that one thing we could consider is adding dynamic help text that describes how to unblock the mass when it is stuck in this case and cannot be moved. @terracoda and @emily-phet both said that we should hold off on this for now and maybe come back to this later.

Was a good point @Nancy-Salpepi, thanks for pointing out. Closing.
